httpclient-vertx/src/main/java/io/fabric8/kubernetes/client/vertx/VertxHttpClient.java

```diff
@@ -122,6 +134,9 @@ public CompletableFuture<HttpResponse<AsyncBody>> consumeBytesDirect(StandardHtt
   @Override
   public void doClose() {
     client.close();
```

Comment:
Just in case, a try finally would be good here.

Reply:
I'm not completely following, do you mean like this:
```
    try {
      client.close();
    } finally {
      if (closeVertx) {
        vertx.close();
      }
    }
```